app/models/plan_year.rb

```diff
@@ -455,7 +455,9 @@ def application_errors
     errors
   end
 
-
+  def documents_uploaded?
```

Comment:
you can see how documents_uploaded? => true when it actually needs documents to be uploaded is misleading, right?

the method name needs to be changed to something that indicates what it actually returns and why. 

def no_documents_uploaded? or def attestation_invalid? something like that